class.jetpack-gutenberg.php

```diff
@@ -109,7 +109,7 @@ public static function should_load_blocks() {
 	 *
 	 * @return void
 	 */
-	public static function load_assets_as_required( $type ) {
+	public static function load_assets_as_required( $type, $dependencies = null ) {
```

Comment:
Can we rename `$dependencies` => `$script_dependencies`.

Reply:
And also update the doc block above.

Reply:
Both resolved in https://github.com/Automattic/jetpack/pull/10503/commits/ea47243da7b3ed3ec8e0201ff7a237638f807163